lambdas/src/test/kotlin/scorcerer/utils/PointsCalculatorTest.kt

```diff
@@ -0,0 +1,24 @@
+import io.kotlintest.shouldBe
+import org.junit.jupiter.api.DynamicTest
+import org.junit.jupiter.api.TestFactory
+import scorcerer.db.Prediction
+import scorcerer.utils.PointsCalculator
+import scorcerer.utils.Result
+
+internal class PointsCalculatorTest {
+    data class TestInput(val prediction: Prediction, val result: Result)
+
+    @TestFactory
+    fun testCalculatePoints() =
```

Comment:
Can we not just used @ParameterizedTest here instead, i'm not fully convinced by dynamic tests